utils/dataset_utils.py

```diff
@@ -72,6 +73,8 @@
 
 _MAX_ROWS = 200000
 
+logs = utils.prepare_logging(__file__)
```

Comment:
@TristanThrush , in order to write to logs in this file, you'll have to initialize it; set it up here.